ephios/event_management/signup.py

```diff
@@ -86,7 +86,7 @@ class LocalUserParticipant(AbstractParticipant):
     user: get_user_model()
 
     def create_participation(self, shift):
-        return LocalParticipation.objects.create(shift=shift, user=self.user)
+        return LocalParticipation(shift=shift, user=self.user)
```

Comment:
maybe rename the method? could be misleading because now the participation is not persisted after creation